### PR TITLE
The image details are not hidden when click on same image

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#690 magento/adobe-stock-integration/issues/690

### Manual testing scenarios (*)

1. Login to admin panel
2. Navigate to Cms Pages
3. User clicks Add New Page button
4. Expand "Content" section
5. User clicks "Insert Image..." button
6. Click "Search Adobe Stock" button to open images grid
7. Click on an image to expand the preview

Expected result:
User sees that image details are hidden
### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-6285](https://jira.corp.magento.com/browse/ENGCOM-6285) has been created to process this Pull Request
:eight_spoked_asterisk: @sivaschenko, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

File: app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js [hunk 1]
```diff
@@ -101,6 +101,13 @@ define([
         show: function (record) {
             var img;
 
+            if (record._rowIndex === this.visibleRecord() &&
+                this.isVisible(record)) {
```
Comment (sivaschenko) [inline app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js]: There is no need to call `this.isVisible(record)` as the first condition seems to be enough

File: app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js [hunk 2]
```diff
@@ -101,6 +101,13 @@ define([
...
+                this.hide();
```
Comment (sivaschenko) [inline app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js:105]: Let's just move the codition below the `this.hide();` not to duplicate the call

**Review by sivaschenko — CHANGES_REQUESTED** [review]

File: app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js (hunk 1 above)
Comment (Nazar65) [reply app/code/Magento/Ui/view/base/web/js/grid/columns/image-preview.js]: ok

**Review by sivaschenko — APPROVED** [review]

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-6285](https://jira.corp.magento.com/browse/ENGCOM-6285) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
Hi @engcom-Golf . Image still cannot be hidden by clicking on it in images grid when image preview was opened.
Could you take a look?

**Comment by engcom-Golf** [comment]
Hi @engcom-Delta thank you for attention on this pr, yes this issue related to the latest changes where method `hide()` moved to the beginning of if statement, so we always have `visibleRecord(null)` so I think the best way is to revert and place `hide()` method inside of if statement and boby of the method, as `hide()` method sets null to all observables.

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-6285](https://jira.corp.magento.com/browse/ENGCOM-6285) has been created to process this Pull Request

**Comment by engcom-Delta** [comment]
@engcom-Golf There is issue with closing image preview when user clicks on the image on the next page that has same index in grid as original image
**Steps to reproduce:**

- Login to admin panel
- Navigate to Cms Pages
- User clicks Add New Page button
- Expand "Content" section
- User clicks "Insert Image..." button
- Click "Search Adobe Stock" button to open images grid
- Click on an first image to expand the preview on first page
- Change grid page.
- Click on an first image on new page.

:x: **Actual result:**
Image details are hidden

#25566PR

**Comment by sivaschenko** [comment]
@VasylShvorak can you please report it as a separate issue to Adobe Stock Integration project: Image preview should be closed when the page is changed

**Comment by engcom-Delta** [comment]
:heavy_check_mark: QA passed

**Pull request merged**